src/webapi/event.rs

```diff
@@ -453,3 +1085,55 @@ reference_boilerplate! {
     instanceof Event
     convertible to Event
 }
+
+impl ErrorEvent {
+    /// Returns a human-readable error message describing the problem.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/message)
+    #[inline]
+    pub fn message( &self ) -> String {
+        return js!(
+            @{self.as_ref()}.message;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the name of the script where the error occurred.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/filename)
+    #[inline]
+    pub fn filename( &self ) -> String {
+        return js!(
+            @{self.as_ref()}.filename;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the line number of the script file where the error occurred.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/lineNo)
+    #[inline]
+    pub fn line_no( &self ) -> i32 {
+        return js!(
+            @{self.as_ref()}.lineno;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the column number of the script file where the error occurred.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/colNo)
+    #[inline]
+    pub fn col_no( &self ) -> i32 {
+        return js!(
+            @{self.as_ref()}.colno;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the object that is concerned with the event.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent/error)
+    #[inline]
+    pub fn error( &self ) -> Reference {
```

Comment:
MDN tells me this is an experimental API.

I'm not against including experimental APIs, but I'd be nice to at least document that in the docs. (e.g. put `(experimental API)` or something like that on the first line so that the user immediately knows it's experimental)